### PR TITLE
Milestone 3: Mobile version contact form

### PR DESCRIPTION
**Pull request by BenMKT**
- Add a contact form to my portfolio.
- Name field accepts up to 30 chars.
- Email field accepts only emails.
- Message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
- Added required attribute.
- Collect data using the [Formspree service](https://formspree.io/html).
- Used Flexbox for layout.

- Implemented the button interactions (enable, hover, etc.).
- Used responsive values in my CSS rules instead of pixels.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -148,5 +147,22 @@ <h2>About Myself</h2>
             </div>
         </section>
     </main>
+    <footer class="contact">
```
Comment (zieeco) [inline index.html]: - Hey Team, great with the page layout and the form section implementation, however, this container should be a section tag and not a `footer` as this particular Figma design doesn't have a footer and as such, this should another section of your page. To improve our semantic usage, kindly update this to a section tag.

File: index.html [hunk 2]
```diff
@@ -148,5 +147,22 @@ <h2>About Myself</h2>
...
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that
+                needs
+                coding.I’d love to help with it!</p>
+            <form action="https://formspree.io/f/xpzeyzyd" method="post">
+                <label for="fname"><input type="text" name="userName" id="fname" maxlength="30" placeholder="Full Name"
+                        required></label>
+                <label for="email"><input type="email" id="email" name="userEmail" placeholder="Email" required></label>
+                <label for="messages"><textarea name="message" maxlength="500" id="messages"
+                        placeholder="Write your message here" cols="30" rows="10" required></textarea></label>
+                <button type="submit">Get in touch</button>
```
Comment (zieeco) [inline index.html:162]: - Also, the button is missing the relevant colors, and background color when hovered, active, enabled, and disabled. kindly update this to improve the user experience of your website.

image

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Form works correctly.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the co

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (BenMKT) [reply index.html]: - Changed the `footer` tag to a `section` tag for container and replaced the former `section` tag with a `div` tag.

File: index.html (hunk 2 above)
Comment (BenMKT) [reply index.html:162]: - Added background color on hover and when active states.

**Review by stanleySimeon — APPROVED** [review]
# Hi @BenMKT, @zieeco, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
- Meaningful commit messages ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**